react/ModalContext.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { createContext } from 'react'
+
+const { Consumer, Provider } = createContext({})
+
+interface ModalContext {
+  closeModal: () => void
```

Comment:
No `openModal`?

Reply:
@luciannojunior I think this context will only be applied to children of the modal, so from their point of view, the modal has always been open (I might be mistaken though)

Reply:
I agree with @lbebber and I don't think openModal should be included, at least for now.